2021/11/04 12:42:07 | python_logging.py(83 行) | INFO| 开始记录执行开始时间-------
2021/11/04 12:42:08 | python_logging.py(83 行) | INFO| fun1执行一共花费0.013643026351928711秒-------
2021/11/04 12:42:08 | python_logging.py(83 行) | INFO| fun1函数执行结果为3
2021/11/04 12:42:08 | python_logging.py(83 行) | INFO| 开始记录执行开始时间-------
2021/11/04 12:42:09 | python_logging.py(83 行) | INFO| fun2执行一共花费0.008403539657592773秒-------
2021/11/04 12:42:09 | python_logging.py(83 行) | INFO| fun2函数执行结果为12
2021/11/04 12:42:09 | python_logging.py(83 行) | INFO| 开始记录执行开始时间-------
2021/11/04 12:42:10 | python_logging.py(87 行) | ERROR| 执行fun3函数过程中出错，错误原因为can't multiply sequence by non-int of type 'str'
